ui/components/MesheryResultDialog.js

```diff
@@ -161,52 +158,48 @@ class MesheryResultDialog extends React.Component {
         </Typography>
         <Table className={classes.table} size="small" aria-label="Load Profile">
           <TableBody>
-            {/* {this.createTableRow('Name', rowData.name)} */}
+            {/* {createTableRow('Name', rowData.name)} */}
```

Comment:
Comment can be removed if there is not use of it, or it should mention why this part is commented

Reply:
Yes, the comment can be removed as the line depicts its functionality
```
title={`Details${rowData ? ` - ${rowData.name}` : ''}`}
```